cmd/root_test.go

```diff
@@ -76,6 +78,65 @@ func TestRootCommand(t *testing.T) {
 	}
 }
 
+func TestExecuteContext(t *testing.T) {
```

Comment:
note: I didn't use the table test format because I think the future Interrupt and Cancelled Context tests will need a custom format. However, I can refactor this to the table test format if the reviewers prefer.

Reply:
No problem, I think it's fair to optimize the tests and reduce the duplicate code. Commit [5e57be4](https://github.com/slackapi/slack-cli/pull/61/commits/5e57be43e50db68ac0503773da63beb2cc7e3e66) rewrites the tests as a table test (and it's much shorter!)